src/Tribe/Views/V2/Service_Provider.php

```diff
@@ -28,6 +28,7 @@ public function register() {
 		}
 
 		require_once tribe( 'tec.main' )->plugin_path . 'src/Tribe/Views/V2/functions/template-tags.php';
+		require_once tribe( 'tec.main' )->plugin_path . 'src/Tribe/Views/V2/functions/classes.php';
```

Comment:
:no_entry_sign: Local non-PHP file should be loaded via `file_get_contents` rather than via `require_once`.

<sub>hash: b9967213d322547751412e7cdab2efb7</sub>